src/components/Icon.tsx

```diff
@@ -28,7 +28,6 @@ type Props = IconProps & {
   /**
    * @optional
    */
-  theme: Theme;
 };
 
 const isImageSource = (source: any) =>
```

Comment:
here I have an error to show the icon as react-native-paper.
Example
 <SettingsConsumer>
        {({ icon }) => {
          return icon({
            icon: s,
            color: iconColor,
            size,
            direction,
          });
        }}
      </SettingsConsumer>
@joelmsanto